system_tests/seqfeed_test.go

```diff
@@ -144,8 +147,36 @@ func TestRelayedSequencerFeed(t *testing.T) {
 	}
 }
 
+func comparesMsgResultFromConsensusAndExecution(
```

Comment:
This is pretty specific to tests with only 1 message (like where it's used).
Let's have a version that will loop through all messages and compare exec with consensus node functions. Add it to the end of every test that has a reorg.

Reply:
Done.